README.md

```diff
@@ -1,2 +1,55 @@
-# pharmacy-data-etl
-ETL to retrieve pharmacy data from NHS Organisations API
+# Pharmacy Data ETL
+ETL to retrieve Pharmacy information from [NHS Organisation API](http://api.nhs.uk/organisations) based on listings in [NHS Choices Syndication](http://www.nhs.uk/aboutNHSChoices/professionals/syndication/Pages/Webservices.aspx)
+and store as JSON.
+
+## Run process
+
+In order for the process to access the syndication feed an API key is required.
```

Comment:
An Azure key is also required but is not covered here.  Maybe point out the mandatory env vars here and cover it in more detail in the Environment variables section.